src/lib/pages/backoffice/Actions/CheckOut/CheckOutSearch/index.js

```diff
@@ -0,0 +1,26 @@
+import { connect } from 'react-redux';
+import CheckOutSearchComponent from './CheckOutSearch';
+import {
+  checkOutSearch,
+  clearSearch,
+  updateQueryString,
+} from '../state/actions';
+
+const mapDispatchToProps = dispatch => ({
+  checkOutSearch: term => dispatch(checkOutSearch(term)),
+  updateQueryString: qs => dispatch(updateQueryString(qs)),
```

Comment:
is there any reason that it is a redux state, not internal component state?

Reply:
Great question in first place. Few of the reasons, not really strong opinion.

1) Since the state of the CheckOut reducer is based on that string it makes "logical' sense to keep it as part of the what caused that state.
2) It plays well with other state changes like clear results without mixing presentation logic with bussiness.
3) Matches the implementation of CheckIn feature.

We can of course move it in the component state but would be more messy IMHO, wdyt?